mdp/nodes/recursive_expansion_nodes.py

```diff
@@ -0,0 +1,623 @@
+"""
+This module provides nodes for recursive computation of sequences of functions.
+"""
+import mdp
+from mdp import numx as np
+from mdp.nodes import PolynomialExpansionNode
+__docformat__ = "restructuredtext en"
+
+
```

Comment:
Put the functions below into consistent order, i.e. always
init_x
recf_x

or always

recf_x
init_x.
Currently it's someimes init first, sometimes recf first.

Reply:
Fixed.